src/Components/Users/UserProfile.tsx

```diff
@@ -581,7 +784,7 @@ export default function UserProfile() {
             Check for software updates
             <p className="mt-1 text-sm leading-5 text-gray-600">
               Click the update button to see if you have the latest
-              &quot;care&quot; version.
+              &quotcare&quot version.
```

Comment:
why was this line changed?
![image](https://user-images.githubusercontent.com/29787772/180138210-403d89d8-1eb4-43e9-9d14-e400193283f2.png)

Reply:
yeah, I've fixed that.